### ISSUE TITLE
Set up test coverage badge

### ISSUE THREAD

**Issue by Apanatshka** [BurntSushi/cargo-benchcmp#12]
I saw [another crate](https://github.com/phsym/prettytable-rs) use a badge for test coverage using a service called [Coveralls](https://coveralls.io). I have no experience with them, but it looks nice. Might be a good idea when #3 is resolved?

**Comment by BurntSushi** [comment]
Yes please! I think lifting this straight from their `.travis.yml` will do the trick: https://github.com/phsym/prettytable-rs/blob/master/.travis.yml#L15

**Comment by Apanatshka** [comment]
I just ran `kcov` locally, because I was curious and the unit (quickcheck) tests give a 93.3% coverage :smile: 
I also tried to get coverage from the integration tests but that doesn't seem to be working..

**Comment by BurntSushi** [comment]
Yeah, I don't expect the integration tests to contribute to it automatically. I think you'd have to modify the integration test suite to run `kcov` itself, and then merge the results for every test. It's not worth it.

**Comment by Apanatshka** [comment]
Yeah, I'm not going to look into it. Can you set up the badge? I would do it for you but it's inconvenient to get this working of my fork first then get the values right in a PR.

**Comment by BurntSushi** [comment]
Yup, will do when I get a chance.
